### PR TITLE
Implement adminRestrictedOperation config to support disabling multiple providers

### PR DESCRIPTION
**Pull request by xil222**
test sheet reference:
https://docs.google.com/spreadsheets/d/1zCSRaMbhkZBhXeJ3uX-yqofAxO5Ib1bbkvQnqMniSCU/edit#gid=0

adding GoogleYolo cases

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Implements support for developers to disable sign up on multiple providers in
+FirebaseUI by defining an adminRestrictedOperation config and adding support in
```
Comment (bojeil-google) [inline CHANGELOG.md:2]: This has to be a single line. Change to something more understandable and less technical:
` * Improved UI support for all identity provider sign up operations when they are restricted by site administrators.`

File: soy/elements.soy [hunk 2]
```diff
@@ -25,7 +25,7 @@
   {@param? disabled: bool} /** Whether the email input is disabled or not. */
   {@param? changeEmail: bool} /** Whether to use change email label. */
   <div class="firebaseui-textfield mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-    <label class="mdl-textfield__label firebaseui-label" for="ui-sign-in-email-input">
```
Comment (bojeil-google) [inline soy/elements.soy:28]: You need to revert this change

File: soy/elements.soy [hunk 3]
```diff
@@ -35,7 +35,6 @@
     <input
         type="email"
         name="email"
-        id="ui-sign-in-email-input"
```
Comment (bojeil-google) [inline soy/elements.soy:38]: You need to revert this change

File: soy/elements.soy [hunk 4]
```diff
@@ -64,13 +63,10 @@
     </button>
     <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label{sp}
                 firebaseui-textfield firebaseui-phone-input-wrapper">
-      <label class="mdl-textfield__label firebaseui-label" for="ui-sign-in-phone-number-input">
```
Comment (bojeil-google) [inline soy/elements.soy:67]: You need to revert this change

File: soy/elements.soy [hunk 5]
```diff
@@ -64,13 +63,10 @@
...
+      <label class="mdl-textfield__label firebaseui-label" for="phoneNumber">
         {msg desc="Label of an phone number input field."}Phone number{/msg}
       </label>
-      <input 
-          type="tel" 
-          name="phoneNumber"
-          id="ui-sign-in-phone-number-input"
```
Comment (bojeil-google) [inline soy/elements.soy:73]: You need to revert this change

File: soy/elements.soy [hunk 6]
```diff
@@ -88,13 +84,10 @@
 {template .phoneConfirmationCode}
   <div class="firebaseui-textfield mdl-textfield mdl-js-textfield{sp}
       mdl-textfield--floating-label">
-    <label class="mdl-textfield__label firebaseui-label" for="ui-sign-in-phone-confirmation-code-input">
```
Comment (bojeil-google) [inline soy/elements.soy:91]: You need to revert this change

File: soy/elements.soy [hunk 7]
```diff
@@ -88,13 +84,10 @@
...
+    <label class="mdl-textfield__label firebaseui-label" for="phoneConfirmationCode">
       {msg desc="Label of 6-digit SMS verification code."}6-digit code{/msg}
     </label>
-    <input 
-        type="number"
-        name="phoneConfirmationCode"
-        id="ui-sign-in-phone-confirmation-code-input"
```
Comment (bojeil-google) [inline soy/elements.soy:97]: You need to revert this change

File: soy/elements.soy [hunk 8]
```diff
@@ -329,13 +322,12 @@
 {template .name}
   {@param? name: string} /** The display name to prefill. */
   <div class="firebaseui-textfield mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-    <label class="mdl-textfield__label firebaseui-label" for="ui-sign-in-name-input">
```
Comment (bojeil-google) [inline soy/elements.soy:332]: You need to revert this change

File: soy/elements.soy [hunk 9]
```diff
@@ -329,13 +322,12 @@
...
+    <label class="mdl-textfield__label firebaseui-label" for="name">
       {msg desc="Label of a field for setting the user's name."}First &amp; last name{/msg}
     </label>
     <input
         type="text"
         name="name"
-        id="ui-sign-in-name-input"
```
Comment (bojeil-google) [inline soy/elements.soy:338]: You need to revert this change

File: soy/elements.soy [hunk 10]
```diff
@@ -355,7 +347,7 @@
   {@param? label: string} /** The placeholder text in the field, defaulting to "Choose password". */
   <div class="firebaseui-new-password-component">
     <div class="firebaseui-textfield mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-      <label class="mdl-textfield__label firebaseui-label" for="ui-sign-in-new-password-input">
```
Comment (bojeil-google) [inline soy/elements.soy:358]: You need to revert this change

File: soy/elements.soy [hunk 11]
```diff
@@ -365,7 +357,6 @@
       <input
           type="password"
           name="newPassword"
-          id="ui-sign-in-new-password-input"
```
Comment (bojeil-google) [inline soy/elements.soy:368]: You need to revert this change

File: soy/elements.soy [hunk 12]
```diff
@@ -387,7 +378,7 @@
 {template .password}
   {@param? current: bool} /** Whether to show "Current password" or "Password" label. */
   <div class="firebaseui-textfield mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-    <label class="mdl-textfield__label firebaseui-label" for="ui-sign-in-password-input">
```
Comment (bojeil-google) [inline soy/elements.soy:390]: You need to revert this change

File: soy/elements.soy [hunk 13]
```diff
@@ -397,7 +388,6 @@
     <input
         type="password"
         name="password"
-        id="ui-sign-in-password-input"
```
Comment (bojeil-google) [inline soy/elements.soy:400]: You need to revert this change

**Review by bojeil-google — CHANGES_REQUESTED** [review]

**Review by bojeil-google — APPROVED** [review]

**Comment by xil222** [comment]
@googlebot: rescan

**Pull request merged**